homeassistant/components/motion_blinds/__init__.py

```diff
@@ -62,21 +63,29 @@ def stop_motion_multicast(event):
 
     def update_gateway():
         """Call all updates using one async_add_executor_job."""
+        all_available = True
         motion_gateway.Update()
         for blind in motion_gateway.device_list.values():
             try:
                 blind.Update()
             except timeout:
                 # let the error be logged and handled by the motionblinds library
-                pass
+                all_available = False
 
-    async def async_update_data():
+        return all_available
+
+    async def async_update_data(self):
         """Fetch data from the gateway and blinds."""
         try:
-            await hass.async_add_executor_job(update_gateway)
+            all_available = await hass.async_add_executor_job(update_gateway)
         except timeout:
             # let the error be logged and handled by the motionblinds library
-            pass
+            all_available = False
+
+        if all_available:
+            self.update_interval = UPDATE_INTERVAL
+        else:
+            self.update_interval = 60
```

Comment:
Consider giving this one a descriptive name too like: UPDATE_INTERVAL_FAST_RETRY

Reply:
done